topologytest/couchbase_lite_mock_peer_test.go

```diff
@@ -133,6 +133,7 @@ func (p *CouchbaseLiteMockPeer) WaitForDocVersion(dsName sgbucket.DataStoreName,
 		if !assert.NotNil(c, actual, "Could not find docID:%+v on %p\nVersion %#v", docID, p, expected) {
 			return
 		}
+		assert.True(c, actual.IsHLVEqual(expected), "Actual HLV does not match expected on %s for peer %s.  Expected: %s, Actual: %s", docID, p, expected.HLV, actual.HLV)
```

Comment:
For debugging this, I had implemented a method like `GoString()` on `HybridLogicalVector` because `%s` will print out pointer values for `PreviousVersions` and also having int representations for `HybridLogicalVector.Value` is nice?

Was this something you felt was helpful?

Reply:
Yes, I used that for some of the failing tests.  I don't think I had any failures at this point so hadn't noticed this wasn't switched over to use %#v - I've made that change in the latest commit.

The logs are still fairly dense to parse when triaging failures, but they do a good job of including everything that's needed to trace the replication/HLV states.